### ISSUE TITLE
Cloud Platform installation file script fails

### ISSUE THREAD

**Issue by ppatierno** [rhiot/rhiot#383]
Hi,
the current rhiot-cloud-platform.sh script fails while trying to pull down the rhiot/spark-standalone image with 0.1.3 tag : 

Pulling repository docker.io/rhiot/spark-standalone
Tag 0.1.3 not found in repository docker.io/rhiot/spark-standalone
Error response from daemon: no such id: spark_master
Error: failed to remove containers: [spark_master]
Unable to find image 'rhiot/spark-standalone:0.1.3' locally
Pulling repository docker.io/rhiot/spark-standalone
Tag 0.1.3 not found in repository docker.io/rhiot/spark-standalone
Error: No such image or container: spark_master
Error response from daemon: no such id: spark_worker
Error: failed to remove containers: [spark_worker]
Unable to find image 'rhiot/spark-standalone:0.1.3' locally
Pulling repository docker.io/rhiot/spark-standalone
Tag 0.1.3 not found in repository docker.io/rhiot/spark-standalone

The Docker Hub doesn't contain the related image with this tag but it has 0.1.4-SNAPSHOT (as latest).
The tag is get using the RHIOT_VERSION=0.1.3 variable.

Paolo.

**Comment by hekonsek** [comment]
Yes, this is a part of the major refactoring I was referring to :) Please use latest snapshot i.e.:  

    $ RHIOT_VERSION=0.1.4-SNAPSHOT bash <(curl -L -s https://goo.gl/nVsrg6)

Cheers!

**Comment by hekonsek** [comment]
BTW I plan to finish refactoring and make our documentation stable again before the 0.1.4 release. Now my priority is to refactor the project to match Eclipse Service Platform architecture [1]. This is our top priority now, as we plan to donate part of our code to the emerging Eclipse Server Platform project in a near future.

[1] https://wiki.eclipse.org/IoT/IoTConnectorProposal

**Comment by ppatierno** [comment]
Ok thanks ! 

I already knew about this proposal ... it's an open source project for developing an architecture like the ones offered by companies like Microsoft (Azure IoT Hub / IoT Suite, IBM (IoT Foundation) and Amazon (AWS IoT).

**Comment by hekonsek** [comment]
Exactly. We are going to align Rhiot to that architecture and work with Eclipse on this new project. This is really cool stuff, but requires us to refactor Rhiot. I'm working on this right now and will be working in a coming days as well.
